tests/playbooks/test_minimal.yaml

```diff
@@ -31,6 +31,8 @@
     - neutron_adoption
     - glance_adoption
     - placement_adoption
+    - nova_adoption
```

Comment:
please add it here also: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/playbooks/test_with_ceph.yaml#L27